example_usage.ipynb

```diff
@@ -38,10 +38,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
```

Comment:
Did you actually change anything in this notebook? If not, could you revert the changes to it? (damn notebooks and git)

Reply:
Not intentionally! Will revert!